src/lib/package.json

```diff
@@ -35,8 +36,8 @@
   ],
   "peerDependencies": {
     "@fortawesome/fontawesome-svg-core": "^1.2.0",
-    "@angular/common": "^6.0.0 || ^7.0.0",
-    "@angular/core": "^6.0.0 || ^7.0.0"
+    "@angular/common": "^6.0.0 || ^7.0.0 || ^8.0.0-rc.2",
```

Comment:
I have checked couple of popular Angular libraries and they seem to only support one major version of Angular at the same time. Let's do the same here. So please remove 6 and 7 from the peer dependencies.